taca/illumina/HiSeqX_Runs.py

```diff
@@ -60,11 +60,12 @@ def demultiplex_run(self):
 
         ssname   = self._get_samplesheet()
         ssparser = SampleSheetParser(ssname)
+        indexfile = self.CONFIG['bcl2fastq']['index_path']
```

Comment:
it is safer to use get and wrap it in a try except so that we can get a nice error and conclusion of the program if index_path is missing.